.cirrus.yml

```diff
@@ -282,13 +282,13 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_mac.sh"
 
 task:
-  name: 'macOS 11 native [gui, sqlite only] [no depends]'
+  name: 'macOS 12 native [gui, system sqlite only] [no depends]'
   brew_install_script:
-    - brew install boost libevent qt@5 miniupnpc libnatpmp ccache zeromq qrencode sqlite libtool automake pkg-config gnu-getopt
+    - brew install boost libevent qt@5 miniupnpc libnatpmp ccache zeromq qrencode libtool automake gnu-getopt
```

Comment:
Why is this change needed and what does it effect?

Reply:
Homebrew's `pkg-config` has already been install in this image.

https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md#wallet-dependencies:
> Apple has included a useable `sqlite` package since macOS 10.14. You may not need to install this package.